content/api/suppression-list.apib

```diff
@@ -165,85 +176,58 @@ Perform a filtered search for entries in your suppression list.
+                    "updated": "2017-01-01T01:01:01+00:00",
                     "transactional": true
                 },
                 {
                     "recipient": "test@example.com",
                     "description": "550: this email address does not exist #55",
                     "source": "Bounce Rule",
                     "type": "non_transactional",
-                    "created": "2015-01-01T01:01:01+00:00",
-                    "updated": "2015-01-01T01:01:01+00:00",
+                    "created": "2017-01-01T01:01:01+00:00",
+                    "updated": "2017-01-01T01:01:01+00:00",
                     "non_transactional": true
                 },
                 {
                     "recipient": "test2@example.com",
                     "description": "Recipient unsubscribed",
                     "source": "Manually Added",
                     "type": "transactional",
-                    "created": "2015-01-01T01:01:01+00:00",
-                    "updated": "2015-01-01T01:01:01+00:00",
+                    "created": "2017-01-01T01:01:01+00:00",
+                    "updated": "2017-01-01T01:01:01+00:00",
                     "transactional": true
                 }
             ],
             "links": [],
             "total_count": 3
         }
 
-+ Response 200 (application/json; charset=utf-8)
-    + Body
-
-            {
-                "results": [
-                    {
-                        "recipient": "test@example.com",
-                        "non_transactional": true,
-                        "type": "non_transactional",
-                        "source": "Bounce Rule",
-                        "description": "550: this email address does not exist #55",
-                        "created": "2015-01-01T01:01:01+00:00",
-                        "updated": "2015-01-01T01:01:01+00:00"
-                    },
-                    {
-                        "recipient": "test@example.com",
-                        "transactional": true,
-                        "type": "transactional",
-                        "source": "Bounce Rule",
-                        "description": "550: this email address does not exist #55",
-                        "created": "2015-01-01T01:01:01+00:00",
-                        "updated": "2015-01-01T01:01:01+00:00"
-                    }
-                ]
-            }
-
-## Retrieve, Delete, Insert or Update [/suppression-list/{recipient_email}]
-
-### Retrieve a Recipient Suppression Status [GET]
 
-Retrieve the suppression status for a specific recipient by specifying the recipient’s email address in the URI path.
+### Retrieve a Suppression [GET /suppression-list/{recipient_email}]
 
-This is a search endpoint so delays can occur when searching across multiple suppression lists.  See "Searching with Subaccounts" below.
+Return all suppression entries for a recipient. If the recipient is not in the suppression list, an HTTP status of 404 is returned.
```

Comment:
"Return all suppression entries for a recipient." sounds like you can have an infinite number of suppression entries when you can really just have two.

Reply:
No it is infinite, 2 per subaccount